redirect.go

```diff
@@ -165,8 +176,11 @@ func (r *Redirect) OldInputs() map[string]string {
 
 // Get old input data by key.
 func (r *Redirect) OldInput(key string) string {
-	msgs := r.c.redirectionMessages
+	if r.c == nil {
```

Comment:
When will this be the case?

Reply:
Missing safety check found by https://github.com/uber-go/nilaway